seamm_datastore/api.py

```diff
@@ -281,3 +288,117 @@ def get_users(_=None, as_json=False):
         users = UserSchema(many=True).dump(users)
 
     return users
+
+
+def finish_job(session, job_id, finish_time, status="finished", as_json=False):
+    """Set the status and time that the job finished.
+
+    Parameters
+    ----------
+    session : session object
+        The SQLAlchemy session
+    job_id : int
+        The ID of the job, eg. 209
+    finish_time : datetime.datetime
+        The time when the job finished.
+    status : str
+        The status, such as "error" or the default, "finished"
+
+    Returns
+    -------
+    bool
+        True if the finish time was successfully set, False otherwise.
+    """
+    from seamm_datastore.database.models import Job
+
+    job = Job.query.filter_by(id=job_id).one_or_none()
+    if job is None:
+        return False
+    else:
+        job.finished = finish_time
+        job.status = status
+        session.commit()
+        return True
+
+
+def submit_job(
```

Comment:
Can you comment on the difference between `add_job` and `submit_job`? Not sure I understand the difference.